### PR TITLE
feat: removed feature flag for confirmations screen

### PR DESCRIPTION
**Pull request by NidhiKJha**
This PR is to remove the feature flag from confirmations page

## **Related issues**

Fixes: [https://github.com/MetaMask/MetaMask-planning/issues/3517](https://github.com/MetaMask/MetaMask-planning/issues/3517)

## **Manual testing steps**

1. Connect extension to a Dapp
2. Try to switch to a non enabled network from Dapp
3. In the confirmations screen, we should not see the network picker
4. The Definition list component should expand and popup should not show up on click of See Details

### **Before**
Screenshot 2024-10-16 at 5 55 15 PM

### **After**
Screenshot 2024-10-16 at 5 54 55 PM

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by sahar-fehri** [comment]
Thanks @NidhiKJha ! Looks good; i have tested this locally; when switching to Base i do see a warning at the top which does not appear in the screenshot just wanted to make sure its expected
Screenshot 2024-10-16 at 19 32 30
Screenshot 2024-10-16 at 19 32 49

**Comment by sahar-fehri** [comment]
Did not have latest changes; tested locally again and i dnt see the warning anymore

**Review by sahar-fehri — DISMISSED** [review]

**Comment by sahar-fehri** [comment]
LGTM! thnx 🙏

**Review by sahar-fehri — DISMISSED** [review]

File: ui/pages/confirmations/confirmation/confirmation.js [hunk 1]
```diff
@@ -552,17 +550,12 @@ export default function ConfirmationPage({
       )}
       <Box
         className="confirmation-page__content"
-        padding={process.env.CHAIN_PERMISSIONS && !isSnapCustomUIDialog ? 4 : 0}
+        padding={4}
```
Comment (pedronfigueiredo) [inline ui/pages/confirmations/confirmation/confirmation.js:553]: Is this change intentional? Can you share the thought process behind it?

**Review by pedronfigueiredo — DISMISSED** [review]

**Review by sahar-fehri — APPROVED** [review]

**Review by pedronfigueiredo — APPROVED** [review]
